src/main/java/org/isf/medicalinventory/gui/MedicalPicker.java

```diff
@@ -0,0 +1,327 @@
+package org.isf.medicalinventory.gui;
```

Comment:
Missing license header.

I am stopping making comments; there are just too many things that need changing.